src/hooks/useCommentForm.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  ChangeEvent,
+  FormEvent,
+  useCallback,
+  useContext,
+  useState,
+} from 'react';
+import { PostsContext } from '../context/PostContext';
+import { Comment } from '../types/Comment';
+
+export const useCommentForm = (
+  handleAddComment: (comment: Omit<Comment, 'id'>) => Promise<void>,
+) => {
+  const { selectedPost } = useContext(PostsContext);
+
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
```

Comment:
can we combine these three states into one, so we don't need three function for updates of name, email, body